eclair-core/src/main/scala/fr/acinq/eclair/balance/ChannelsListener.scala

```diff
@@ -0,0 +1,66 @@
+package fr.acinq.eclair.balance
+
+import akka.Done
+import akka.actor.typed
+import akka.actor.typed.Behavior
+import akka.actor.typed.eventstream.EventStream
+import akka.actor.typed.scaladsl.adapter.ClassicActorRefOps
+import akka.actor.typed.scaladsl.{ActorContext, Behaviors}
+import fr.acinq.bitcoin.ByteVector32
+import fr.acinq.eclair.balance.ChannelsListener.{ChannelData, ChannelDied, Command, GetChannels, GetChannelsResponse}
+import fr.acinq.eclair.channel.Helpers.Closing
+import fr.acinq.eclair.channel.{ChannelPersisted, ChannelRestored, HasCommitments}
+
+import scala.concurrent.Promise
+
+object ChannelsListener {
```

Comment:
The goal of this actor is to maintain a local cache of all channel data, that can be used by the API, instead of querying thousands of channel actors. Surprisingly, this is turns out to not be noticeably faster than the `ask`+`Future.sequence` pattern.

Maybe we should remove this? Or we could keep it and use that for all remaining API calls, because it saves some burden of the channel actors? I'm not sure.

Reply:
I think it's useful to have it, when we'll have more load on the channel actors it should be noticeably useful, don't you think?